### DISCUSSION TITLE
Kurs aufzeichnung => aufgezeichneter Kurs laden?

### DISCUSSION THREAD

**Discussion by DerLaechler** [Courseplay/Courseplay_FS22#973] [Q&A] [answered]
Ich bin begeistert, dass ich wieder einen Kurs aufzeichnen kann. und mir wird auch die Meldung angegeben, Kurs als cp-xy gespeichert, ich finde den Kurs in meinem Modsetting-Ordner. aber im Spiel kann ich ihn nicht laden. gibt es einen Weg diese Kurse zu laden, oder muss ich ihn irgendwie modifizieren - ausserhalb des spiels?

Super Arbeit soweit. Vielen Dnak

**Comment by Tensuko** [comment, marked as answer]
Das ist kein Kurs aufzeichnen, das ist Feldrandkurs aufzeichnen, welchen du auf deiner Karte im AI Menü siehst.
Für mehr ist das nicht da.
